nest/src/issue/issue.controller.ts

```diff
@@ -17,6 +17,7 @@ import { IssueService } from './issue.service';
 
 @Controller({ path: 'issues', version: '1' }) // Set controller prefix and api version for all endpoints in this controller
 @ApiOAuth2([ AdminApiScope ]) 	              // Set Swagger OAuth scope
+@UseInterceptors(CacheInterceptor)            // Add auto-caching for GET endpoints. Can also be done globally, or manually. https://docs.nestjs.com/techniques/caching
```

Comment:
For ryddighetens skyld så burde denne ligge over @Api... dekoratorene siden de hører til swagger 👼🏼

Reply:
Enig, fikset 👍